spec/concepts/todo/create_spec.rb

```diff
@@ -1,7 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Todo::Create do
-  it "can be spec'd" do
-    raise "Implement me"
+  it "Should be persisted" do
+    params = {todo: {title: 'Test Title'}}
+    todo = Todo::Create.(params).model
+
+    expect(todo.persisted?).to be
```

Comment:
great! you could use `.to be_new_record` for persistence testing too. In this case... the expectation should be `false`.

Anyway, nice coding style. :+1: :wink: